packages/devtools_app/lib/src/extension_points/extensions_external.dart

```diff
@@ -53,3 +46,47 @@ class ExternalDevToolsExtensionPoints implements DevToolsExtensionPoints {
   @override
   bool get defaultIsDarkTheme => true;
 }
+
+const _newDevToolsIssueUriDisplay = 'github.com/flutter/devtools/issues/new';
+
+Uri _newDevToolsGitHubIssueUri({String? issueDetails}) {
+  const _maxGitHubUriLength = 8190;
+
+  final issueBody = [
+    if (issueDetails != null) issueDetails,
+    ..._issueLinkDetails(),
+  ].join('\n');
```

Comment:
since there is a max URI length, we should make sure that the `issueLinkDetails` items are always included. So instead of trimming at the end, we should trim the `issueDetails` to fit with the complete content of `issueLinkDetails()`.

Better to cut off the end of a stack trace than the version / connected app information.